### DISCUSSION TITLE
Node Thresholding

### DISCUSSION THREAD

**Discussion by sean-afshar** [talmolab/sleap#1113] [Ideas]
Currently one can not change the node threshold for models through the GUI, only through the API when loading the model. It would be helpful to do this through the GUI, as it greatly impacts the number of nodes that are kept during inference, and hence model performance. Add a fill in option to the GUI would be really helpful.

**Comment by gitttt-1234** [comment]
Closing this discussion, this feature is now added to the GUI in #2704
